src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -4937,4 +4961,66 @@ export class PostgresCohortMembersService {
+      }
+
+      // Build ILIKE conditions for each search term with smart column prioritization
+      const searchConditions = searchTerms.map((term) => {
+        if (!term || typeof term !== 'string') {
+          return "";
+        }
+
+        const escapedTerm = term.replace(/%/g, "\\%").replace(/_/g, "\\_");
+
+        // Check if this looks like an email
+        const isEmail = term.includes('@');
```

ESLint

[error] Replace `········const·isEmail·=·term.includes('@'` with `··········const·isEmail·=·term.includes("@"`

(prettier/prettier)